Formula/pachi.rb

```diff
@@ -17,41 +17,29 @@ class Pachi < Formula
     sha256 cellar: :any_skip_relocation, mojave:         "c88f24dd1e7a267848eab540dc2b0961962825ab6e7088fc24b335159dacf31c"
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "e0fc26989c0cf90b6fa2256e129b0b87993464ec27ad88fefe569abdd9702292"
   end
-
-  resource "patterns" do
-    url "https://sainet-dist.s3.amazonaws.com/pachi_patterns.zip"
-    sha256 "73045eed2a15c5cb54bcdb7e60b106729009fa0a809d388dfd80f26c07ca7cbc"
-  end
-
-  resource "book" do
-    url "https://gnugo.baduk.org/books/ra6.zip"
-    sha256 "1e7ffc75c424e94338308c048aacc479da6ac5cbe77c0df8adc733956872485a"
+  
+  depends_on "caffe"
+  
+  resource "datafiles" do
+    url "https://launchpad.net/~lemonsqueeze/+archive/ubuntu/pachi/+sourcefiles/pachi-go-data/12.70/pachi-go-data_12.70.orig.tar.xz"
```

Comment:
why not host this file in the github release?

Reply:
The dcnn files (detlef54.*) are too big to keep in git, that's why they are missing from the github source release. They are included in the binary releases (windows, linux-static) though, so could get them from github instead of launchpad if that's an issue. The good thing with this launchpad resource is it's got just the datafiles, the bad : it usually takes me a day or two to update it after a release. So could pull everything from github if that's preferrable.

Reply:
So where are they kept inbetween binary releases?